internal/rbd/nodeserver.go

```diff
@@ -563,6 +595,27 @@ func (ns *NodeServer) createStageMountPoint(ctx context.Context, mountPath strin
 	return nil
 }
 
+func getDeviceSize(ctx context.Context, devPath string) (int64, error) {
```

Comment:
how about rewriting this and using the functions from `vendor/k8s.io/kubernetes/pkg/volume/metrics_block.go`:
```go
mp := NewMetricsBlock(devPath)
m, err := mp.GetMetrics()
return metrics.Capacity, nil
```

Reply:
Need to extract the size in bytes.  added now. let see if it works.

Reply:
Yes, of course. I was just too lazy to check how to get the bytes from the capacity :sheep: